test/e2e/testdata/source-data-transform.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: source-data-transform
+spec:
+  vertices:
+    - name: in
+      source:
+        http: {}
+        transformer:
+          container:
+            image: quay.io/numaio/numaflow-go/map-even-odd
```

Comment:
how is `map-even-odd` working here? shouldn't it be `mapT`?

Reply:
You are right. I was using existing mapFn to test transformer because this PR is before my mapT SDK implementation(it's map but still it's a sidecar container). I will create follow up PRs to switch to mapT.